### PR TITLE
Printing a message with numbers divisible by 3 and 5.

### LINKED ISSUE #1: Task description
FizzBuzz

### PR DESCRIPTION
**Pull request by wojciechkostecki**
Closes #1

### CODE REVIEW

File: src/ClassicVersion.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        for (Integer number : numbers) {
+            if (number % 3 == 0) {
+                System.out.println(number + " Fizz");
+            }
+            if (number % 5 == 0) {
+                System.out.println(number + " Buzz");
+            }
+            if (number % 3 == 0 && number % 5 == 0) {
+                System.out.println(number + " FizzBuzz");
```
Comment (rbobko93) [inline src/ClassicVersion.java:14]: In your code, if number is divisible by 3 and 5 you code will print all of the ifs. 
number + Fizz number + Buzz number + FizzBuzz. So this part is wrong.

File: src/StreamVersion.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class StreamVersion {
+    public static void fizzBuzz() {
+        IntStream.rangeClosed(1, 100)
+                .mapToObj(x -> x % 3 == 0 ? (x % 5 == 0 ? x + " FizzBuzz" : x + " Fizz") : (x % 5 == 0 ? x + " Buzz" : x))
```
Comment (rbobko93) [inline src/StreamVersion.java:7]: That is an interesting solution and I would like to go over this exercise on one of our lessons.

Comment (rbobko93) [reply src/StreamVersion.java:7]: This is also stream, so i dont know why you created second method.

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Naming is as usual off but I won't ask you to change it because the whole exercise was about ifs and streams.

File: src/StreamVersion.java (hunk 2 above)
Comment (wojciechkostecki) [reply src/StreamVersion.java:7]: I created the second method because I didn't know how to not display the remaining numbers in this method.

**Review by rbobko93 — APPROVED** [review]

**Pull request merged**